dialogs/simplelink.js

```diff
@@ -1,7 +1,13 @@
 ﻿(function (CKEDITOR) {
     'use strict';
 
-    var containsScheme = /:\/\//;
+    function containsScheme (href) {
+        return /^[a-zA-Z]+:/.test(href);
+    }
+
+    function isEmail (href) {
+        return href.indexOf('@') !== -1;
```

Comment:
No, not until now. I still think it should be fixed, though – finding and using a simple email regex shouldn't be much of an issue? Note that it doesn't only relate to authentication URLs, but can occur elsewhere as well, where scheme isn't neccessarily present. We use it in MailMojo for instance: https://v2.mailmojo.no/lists/1/contacts/anders@e5r.no/edit